### PR TITLE
Babel upgrade in ia-components

### LINKED ISSUE #79: Upgrade to Babel 7
**Is your feature request related to a problem? Please describe.**

Upgrade to Babel 7

Currently, Babel version 6 is installed. The current version of Jest is incompatible with Babel 6 which causes errors on running tests (See #39 for more details) 

**Stakeholders**
@iisa @bfalling

### PR DESCRIPTION
**Pull request by shreyansh23**
**Description**
Follow up to the comment by @iisa, https://github.com/internetarchive/iaux/pull/51#issuecomment-484210312
Splitting PR #51 into individual package upgrade.
This one handles Babel upgrade in ia-components package.
The package.json was already Babel compatible. The only missing things were the presets that were used by `.babelrc`.

> What does this PR achieve? Is it a feature/hotfix/refactor? Does it close an Issue? If so, use `Closes #`.

Partially closes #79 

**Technical**
n/a

**Evidence**
Screenshot from 2019-04-17 23-55-49
Screenshot from 2019-04-18 01-21-17

### CODE REVIEW

File: packages/ia-components/.babelrc [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
-            "targets": {
-              "node": "current"
-            }
-          }
-        ],
-        "@babel/preset-react"
+  "presets": [
+    [
+      "@babel/env",
+      {
```
Comment (iisa) [inline packages/ia-components/.babelrc]: was there any issue with the linting in the previous version? not sure if we drop the first bracket down to the next line or not.

File: packages/ia-components/package.json [hunk 2]
```diff
@@ -21,6 +21,8 @@
         "@storybook/addon-knobs": "^4.1.4",
         "@storybook/react": "^4.1.4",
         "assert": "^1.4.1",
+        "babel-core": "^7.0.0-bridge.0",
```
Comment (iisa) [inline packages/ia-components/package.json]: remove babel-core. we don't need it anymore

File: packages/ia-components/package.json [hunk 3]
```diff
@@ -21,6 +21,8 @@
...
+        "babel-jest": "^23.4.2",
```
Comment (iisa) [inline packages/ia-components/package.json]: jest has been upgraded with the previous PR, let's make sure master is rebased into this branch

File: packages/ia-components/sandbox/jwplayer-comp/.babelrc [hunk 4]
```diff
@@ -1,23 +1,25 @@
...
-    "presets": [
-        [
-          "env", {
-            "targets": {
-              "node": "current"
-            }
-          }
-        ],
-        "react"
+  "presets": [
```
Comment (iisa) [inline packages/ia-components/sandbox/jwplayer-comp/.babelrc]: is this file needed? not sure what `jwplayer-comp` directory is..

File: packages/ia-components/.babelrc (hunk 1 above)
Comment (iisa) [reply packages/ia-components/.babelrc]: also, make sure @babel/env (or preset-env? which is the latest one to use?) is loaded into package.json

File: packages/ia-components/sandbox/jwplayer-comp/.babelrc (hunk 4 above)
Comment (shreyansh23) [reply packages/ia-components/sandbox/jwplayer-comp/.babelrc]: @iisa, the `JWPlayerContainer` component is being used by `packages/ia-components/sandbox/theatres/audio-1/audio-1-br-nav-wav.js` and `packages/ia-components/sandbox/theatres/audio-1/audio-1.js`. 
The plugins and presets specified in `.babelrc`  of `jwplayer-comp` are already specified in the `.babelrc` of `ia-components`. So it is perfectly safe to delete the inner `.babelrc` file.

File: packages/ia-components/.babelrc (hunk 1 above)
Comment (shreyansh23) [reply packages/ia-components/.babelrc]: @babel/env is shorthand for @babel/preset-env. Both are acceptable as per Babel docs.

**Comment by shreyansh23** [comment]
@iisa 
The `ia-components` package is already updated to Babel 7. The only requirement is to delete the redundant `.babelrc` file in 'jwplayer-comp` directory. 
This PR can be closed as not further Babel upgrade is necessary.

File: packages/ia-components/.babelrc (hunk 1 above)
Comment (shreyansh23) [reply packages/ia-components/.babelrc]: > was there any issue with the linting in the previous version? not sure if we drop the first bracket down to the next line or not.

The changes done was the default behaviour of the "Babel upgrade" package used by me. But I think it is perfectly fine to ignore this linting change in this PR.

Comment (shreyansh23) [reply packages/ia-components/.babelrc]: > also, make sure @babel/env (or preset-env? which is the latest one to use?) is loaded into package.json

@iisa yes, the presets were missing in the `package.json`. I have updated the PR with the changes.

**Comment by iisa** [comment]
resolved with #519
